component-library/src/app/app-routing.module.ts

```diff
@@ -91,6 +88,11 @@ const routes: Routes = [
       component: RequestFormComponent,
       title: 'Request-form.Title'
     },
+    {
+      path: 'datePicker-doc',
+      component: DatePickerDocumentationComponent,
+      title: 'DatePicker-doc'
```

Comment:
Can you use a translation key here and for French as similar to the other items there?

Reply:
Fixed! Thanks :)